OWNERS

```diff
@@ -1,9 +1,11 @@
 approvers:
-  - bennyz
-  - bertinatto
```

Comment:
@bertinatto do you want to stay on the approver list?

Reply:
I'm OK being dropped.